src/test/resources/examples/enumExamples/api.yaml

```diff
@@ -33,6 +33,8 @@ components:
           $ref: '#/components/schemas/ExtensibleEnumObject'
         list_enums:
           $ref: '#/components/schemas/ListEnums'
+        nullable_ref:
```

Comment:
It would be good to see another identical property, but added to the required list